test/e2e/network_test.go

```diff
@@ -166,35 +166,16 @@ func testPodDNSResolution(t *testing.T) {
 	// PowerShell cmdlet Resolve-DnsName, but it is not present either.
 	// TODO: Use a compatible container image for the e2e test suite that includes the
 	//  PowerShell cmdlet Resolve-DnsName.
-	t.Run("Valid URL", testCtx.testPodDNSResolutionWithValidUrl)
-	t.Run("Invalid URL", testCtx.testPodDNSResolutionWithInvalidUrl)
-}
-
-// testPodDNSResolutionWithValidUrl verifies that a curl query succeed using a valid URL
-// with resolvable DNS record
-func (tc *testContext) testPodDNSResolutionWithValidUrl(t *testing.T) {
-	winJob, err := tc.createWindowsServerJob("win-dns-tester-valid-url",
+	winJob, err := testCtx.createWindowsServerJob("win-dns-tester",
 		// curl'ing with --head flag to fetch only the headers as a lightweight alternative
 		"curl.exe --head -k https://www.openshift.com",
 		nil)
 	require.NoError(t, err, "could not create Windows tester job")
-	defer tc.deleteJob(winJob.Name)
-	err = tc.waitUntilJobSucceeds(winJob.Name)
+	defer testCtx.deleteJob(winJob.Name)
+	err = testCtx.waitUntilJobSucceeds(winJob.Name)
 	assert.NoError(t, err, "Windows tester job failed")
 }
 
-// testPodDNSResolutionWithInvalidUrl verifies that a curl query fail using an invalid URL
-// with unresolvable DNS record
-func (tc *testContext) testPodDNSResolutionWithInvalidUrl(t *testing.T) {
```

Comment:
`...because it succeeds even if the request does not hit the DNS server.`
Two things:
1) Why does it always succeed? I would think curl would return an error if the site doesnt exist.
2) Is that the only justification for removing this test? Is there any value in fixing this test?